hls4ml/backends/oneapi/oneapi_backend.py

```diff
@@ -312,6 +312,40 @@ def init_conv2d(self, layer):
             'n_partitions', 1
         )  # TODO Not used yet as there is no codegen implementation of CNNs for oneAPI backend
 
+
+    @layer_optimizer(SeparableConv2D)
```

Comment:
This may never get called, but am not 100% positive. I think the init calls happen after seperable is already split to depthwise and regular convolution (not yet pointwise).